PL2/PL2-launcher/src/main/java/nl/tudelft/pl2016gr2/launcher/Dnav.java

```diff
@@ -57,11 +57,12 @@ private void insertData(RootLayoutController controller) {
 
     // abusing NWKReader class as this class' classloader can access the correct resource
     Reader reader = new InputStreamReader(
-        FullGfaReader.class.getClassLoader().getResourceAsStream("340tree.rooted.TKK.nwk"));
+        FullGfaReader.class.getClassLoader().getResourceAsStream("10tree_custom.rooted.TKK.nwk"));
     BufferedReader br = new BufferedReader(reader);
     TreeParser tp = new TreeParser(br);
 
-    tree = tp.tokenize("340tree.rooted.TKK");
+    tree = tp.tokenize("10tree_custom.rooted.TKK");
+    
```

Comment:
I'm not sure we should keep committing when we switch between data sets. these lines should be kept as is. Could you unstage them from the commit that added them?